source/install/client/install-client-docker.rst

```diff
@@ -0,0 +1,100 @@
+:orphan:
+
+.. include:: /.res/replace.txt
+
+.. _install-client-docker:
+
+######################################################################
+Installing PMM Client via Docker
+######################################################################
+
+Docker images of PMM Client are stored at the `percona/pmm-client`_
+public repository. The host must be able to run Docker 1.12.6 or later,
+and have network access.
+
+Make sure that the firewall and routing rules of the host do not constrain
+the Docker container. For more information, see :ref:`troubleshoot-connection`.
+
+For more information about using Docker, see the `Docker Docs`_.
+
+**********************************************************************
+Setting Up a Docker Container for PMM Client
+**********************************************************************
+
+A Docker image is a collection of preinstalled software which lets you
+run a selected version of PMM Client.
+A Docker image is not run directly.
+You use it to create a Docker container for your PMM Client.
+When launched, the Docker container gives access to the whole functionality
+of PMM Client.
+
+- The setup begins by pulling the required Docker image.
+- Next, you create a special container for persistent |pmm| data.
+- Finally, you create and launch the PMM Client container.
+
+======================================================================
+Pulling the PMM Client Docker Image
+======================================================================
+
+To pull the latest version from Docker Hub:
+
+.. code-block:: bash
+
+   docker pull percona/pmm-client:2
```

Comment:
it should be perconalab, as I see we haven't made any official release to percona repo in dockerhub

Reply:
These docs are in preparation for release. They will be hidden until then. I assume when released it will be from hub.docker?